src/com/owncloud/android/lib/resources/users/GetRemoteUserInfoOperation.java

```diff
@@ -131,9 +131,7 @@ protected RemoteOperationResult run(OwnCloudClient client) {
                 if (userInfo.getQuota() == null) {
                     userInfo.setQuota(new Quota());
                     userInfo.getQuota().setQuota(QUOTA_LIMIT_INFO_NOT_AVAILABLE);
-                }
-                
-                if (userInfo.getQuota().getQuota() == 0) {
+                } else if (userInfo.getQuota().getQuota() == 0) {
```

Comment:
@tobiasKaminsky please review, this is how I resolved the merge conflict while rebasing and this is imho better than "a second simple if by handling it in a else if"

Reply:
:+1: